components/base/dropdown/VDropdownSub.vue

```diff
@@ -0,0 +1,229 @@
+        tempPosition.left =
+            left.value +
+            scrollX.value -
+            (props.invertAlign ? menuSize.width.value - width.value : 0);
+    } else {
+        tempPosition.top = top.value + height.value + scrollY.value;
+        tempPosition.left =
+            left.value +
+            scrollX.value -
+            (props.invertAlign ? menuSize.width.value - width.value : 0);
+    }
+
+    // Apply the menu size to the position to get the right size
+    tempPosition.bottom =
+        windowHeight.value - tempPosition.top - menuSize.height.value - 5;
+    tempPosition.right = windowWidth.value - tempPosition.left - menuSize.width.value;
+
+    // Check if the menu is out of bounds and adjust it
+
+    if (tempPosition.left > windowWidth.value) {
+        tempPosition.left = windowWidth.value - menuSize.width.value;
+    }
+
+    if (tempPosition.left < 0) {
+        tempPosition.left = 0;
+    }
+
+    if (tempPosition.right > windowWidth.value) {
+        tempPosition.right = windowWidth.value - menuSize.width.value;
+    }
+
+    if (tempPosition.right < 0) {
+        tempPosition.right = 0;
+    }
+
+    if (tempPosition.top > windowHeight.value) {
+        tempPosition.top = windowHeight.value - menuSize.height.value;
+    }
+
+    if (tempPosition.top < 0) {
+        tempPosition.top = 0;
+    }
+
+    if (tempPosition.bottom > windowHeight.value) {
+        tempPosition.bottom = windowHeight.value - menuSize.height.value;
+    }
+
+    if (tempPosition.bottom < 0) {
+        tempPosition.bottom = 0;
+    }
+
+    return {
+        left: tempPosition.left + "px",
+        top: tempPosition.top + "px",
+        bottom: tempPosition.bottom + "px",
+        right: tempPosition.right + "px",
+    };
+});
+
+defineExpose({
+    ...dropdown,
+});
+
+useEventListener(scrollableParent, "transitionend", () => {
+    update();
+});
+
+onMounted(() => {
+    update();
+});
+</script>
+
+<template>
+    <div
+        ref="dropdownElement"
+        :class="[
+            props.icon && 'is-dots',
+            props.modern && 'is-modern',
+            props.spaced && 'is-spaced',
+        ]"
+        class="dropdown">
+        <slot name="button" v-bind="dropdown">
+            <VDropdownItem ref="subMenu" :icon="props.icon" @click="dropdown.toggle()">
+                <slot></slot>
+            </VDropdownItem>
+        </slot>
+
+        <Teleport v-if="dropdown.isOpen" to="body">
+            <div class="dropdown-menu" role="menu" tabindex="0" :style="position">
+                <div ref="dropdownMenuElement" class="dropdown-content">
+                    <slot name="content" v-bind="dropdown"></slot>
+                </div>
+            </div>
+        </Teleport>
+    </div>
+</template>
+
+<style lang="scss">
+@import "bulma/sass/utilities/all";
+
+.dropdown-menu {
+    display: block !important;
+    position: absolute;
+    z-index: 100 !important;
+    overflow-y: auto;
+    overflow-x: auto;
+
```

Comment:
```suggestion
    width: fit-content;
    
```

J'ai besoin de ça sinon j'ai des bars de scrolls.
![Screenshot from 2023-09-14 10-43-36](https://github.com/add-eus/library/assets/9000424/6daaa2e1-3200-4209-a6aa-edb682dae6a3)